lib/dl_file_uploader_api_lib/dl_file_uploader_api_lib/schemas/files.py

```diff
@@ -26,8 +26,10 @@ def validate_docs_data(data):
     if not ((data["public_link"] is None) ^ (data["private_path"] is None)):
         raise ValueError("Expected exactly one of [`private_path`, `public_link`] to be specified")
     if data["public_link"] is None:
-        if data["private_path"] is None or data["oauth_token"] is None:
-            raise ma.ValidationError("Both path and token must be provided for private files")
+        if data["private_path"] is None:
+            raise ma.ValidationError("'private_path' must be provided for private files")
```

Comment:
This code will never run
This case is covered by the very first condition (`not (A ^ B)`)